runtime/hydradx/src/xcm.rs

```diff
@@ -185,7 +212,7 @@ parameter_type_with_key! {
 	pub ParachainMinFee: |location: MultiLocation| -> Option<u128> {
 		#[allow(clippy::match_ref_pats)] // false positive
 		match (location.parents, location.first_interior()) {
-			(1, Some(Parachain(ASSET_HUB_PARA_ID))) => Some(50_000_000),
+			(1, Some(Parachain(ASSET_HUB_PARA_ID))) => Some(150_000_000),
```

Comment:
doesn't this break XCM UI? @nohaapav @green-jay

Reply:
Btw, the increase was necessary as the weight was not enough to successfully send XCM to AH.

Acala uses the same value, but they have this stored in their storage instead of hardcoded.

Reply:
> doesn't this break XCM UI? @nohaapav @green-jay

i don't think it does but it still probably should be reverted as this would actually allow sending DOT to AH which we don't want?

Reply:
reverted!